Planet/Settings/PlanetSettingsGeneralView.swift

```diff
@@ -164,6 +164,19 @@ struct PlanetSettingsGeneralView: View {
         if FileManager.default.fileExists(atPath: planetURL.path) {
             useAsExistingLibraryLocation = true
         }
+        // prompt to user when existing planet library location found:
+        if useAsExistingLibraryLocation {
+            let alert = NSAlert()
+            alert.messageText = "Existing Planet Library Found"
+            alert.alertStyle = .warning
+            alert.informativeText = "Would you like to use new library location at: \(url.path), current database including following planets will be replaced with contents at this location."
```

Comment:
As we discussed, possible scenarios including:

- Use the data already found in the new location
- Use the data in existing location to overwrite
- Merge the data found in the existing and the new locations
- Cancel, do nothing